client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -216,12 +216,14 @@
                                         </li>
                                         <li role="menuitem">
                                             <button class="btn btn-link" type="button"
+                                                    data-ng-if="element.type !== 'MONGODB'"
```

Comment:
In those cases, I found that to allow only elements that we know of ex```"element.type === 'MYSQL' || element.type === 'postgresql' || element.type === 'MARIADB'``` instead of denying to certain elements, is a better and clearer way for future developement.